### PR TITLE
DAISY-1054 & DAISY-992 Activation API

### PR DESCRIPTION
**Pull request by doktor500**
@ms-underwriteme

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -79,6 +79,11 @@ append legals/adviser-declaration
 append legals/client-declaration
 append legals/insurer-marketing-preference
 append legals/third-party-marketing-preference
+append legals/provider-legal-confirmation
 
 append message/overview
 append message/list
+
+append activation/overview
```
Comment (ms-underwriteme) [inline build.sh]: Lets put Activation section before Message section.

File: content/activation/create.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+## Create [/api/application/{applicationId}/activation]
+Activate Application. Lists all Basket Items for Application.
+
++ Parameters
+
+    + id (required, string, `1502181407123020689`) ... Unique ID of existing Application.
+
+### Activation of existing Application [PUT]
```
Comment (ms-underwriteme) [inline content/activation/create.md]: Lets call this section Request Activation. Similar to Comparison API sections.

Comment (ms-underwriteme) [inline content/activation/create.md]: I would add more examples, especially the ones that return 400 status for different Application states.

File: content/activation/create.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
+## Create [/api/application/{applicationId}/activation]
```
Comment (ms-underwriteme) [inline content/activation/create.md]: Would call this section List similar to Comparison API sections.

Comment (ms-underwriteme) [inline content/activation/create.md]: We need a section describing the structure of Activation JSON response. Similar to the one in Comparison API.

Comment (doktor500) [inline content/activation/create.md]: I don't understand these two comments, we already have a list.md file, and it looks like we have a section in the create.md file to describe the structure of the Activation JSON response

**Pull request merged**